### ISSUE TITLE
Newest version (0.21.2 & 0.21.2-rc.4) of perceval returns empty table for mbox file

### ISSUE THREAD

**Issue by leilani-reich** [chaoss/grimoirelab-perceval#810]
Hello,

While working on another project, I noticed when running the following command in terminal on Perceval version 0.21.2 and 0.21.2-rc.4, Perceval wasn't able to retrieve any mbox messages and just returned an empty table:

```
/path/to/perceval mbox <mbox_uri> <mbox_path> --json-line
```

In the above command, <mbox_uri> is a variable representing the mbox_path without the ".mbox" at the end.
And <mbox_path> is a variable representing the actual path to the mbox.

So for instance, the command I used was:

```
~/github/ics496/kaiaulu/my_env/bin/perceval mbox thrift-dev thrift-dev.mbox --json-line
```

For the command, Perceval returns this error:

```
[2023-01-28 21:21:16,196] - Sir Perceval is on his quest.
[2023-01-28 21:21:16,198] - Looking for messages from 'thrift-dev' on 'thrift-dev.mbox' since 1970-01-01 00:00:00+00:00 until None
[2023-01-28 21:21:16,198] - Error!: <<class 'NoneType'>> object is not a valid date
[2023-01-28 21:21:16,198] - Sir Perceval completed his quest.
```

This error was the case with Perceval version 0.21.2 and 0.21.2-rc.4 and perhaps other newer versions. 
However, with Perceval version 0.12.24, the mbox messages are being extracted correctly.

Here is the link to download the mbox file that was used: https://cdn.lfdr.de/stmc/ieee_tse_data/mail/thrift-dev.mbox

And here is a link to further information on the issue I was having: https://github.com/sailuh/kaiaulu/issues/145

Thank you

**Comment by sduenas** [comment]
Thanks for reporting this. Apparently, perceval is not getting a default value for `--to-date` parameter. As a temporal fix, you can add to your command the argument `--to-date` with `2100-01-01` as a date (or even a bigger value).

For example:

```
/path/to/perceval mbox <mbox_uri> <mbox_path> --json-line --to-date `2100-01-01`
```

**Comment by leilani-reich** [comment]
This works well, thank you!

**Comment by sduenas** [comment]
This was already fixed by #812 .
